loopchain/blockchain/candidate_blocks.py

```diff
@@ -101,7 +105,10 @@ def add_vote(self, vote: BlockVote):
         else:
             for block in self.blocks.values():
                 if block.height == vote.block_height:
-                    block.add_vote(vote)
+                    try:
+                        block.add_vote(vote)
+                    except VoteError as e:
+                        util.logger.info(e)
```

Comment:
try to catch `VoteError` here?
It is already catched in `CandidateBlock.add_vote`